pkgs/development/python-modules/portolan/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, isPy3k
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "portolan";
+  version = "1.0.1";
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "fitnr";
+    repo = pname;
+    rev = "v"+version;
+    sha256 = "sha256-zKloFO7uCLkqgayxC11JRfMpNxIR+UkT/Xabb9AH8To=";
+  };
+
```

Comment:
```suggestion

  # has no tests
  doCheck = false;

```

Reply:
Staged, will push as soon as everything is ready.